src/assets/scripts/client/aircraft/AircraftCommander.js

```diff
@@ -653,15 +653,25 @@ export default class AircraftCommander {
             return [false, readback];
         }
 
+        // TODO: move somewhere else
```

Comment:
I am looking for a better place for this code... but couldn't find one. Open for suggestions.

Reply:
moved to ScoreLogic